spec/spec_helper.rb

```diff
@@ -16,7 +16,13 @@
 # users commonly want.
 #
 # See http://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+
 RSpec.configure do |config|
+  config.after(:each, type: :controller) do
+    SmarfDoc.run!(request, response)
+  end
+
+  config.after(:suite) { SmarfDoc.finish! }
```

Comment:
FYI, this will run every time your un your tests. We tend to keep it commented out and then regenerate the docs on purpose when we need to.